event_routing_backends/backends/events_router.py

```diff
@@ -28,7 +28,95 @@ def __init__(self, processors=None, backend_name=None):
         self.processors = processors if processors else []
         self.backend_name = backend_name
 
-    def send(self, event):
+    def configure_host(self, host, router):
```

Comment:
This function is just broken out from the original send() method so both send() and bulk_send() can use it, I don't believe anything changed here.